src/DBManager.cpp

```diff
@@ -4,11 +4,12 @@
 #include <iostream>
 #include <sstream>
 #include <unordered_map>
+#include <utility>
 
 namespace fs = boost::filesystem;
 using namespace std;
 
-DBManager::DBManager(const string& db_path) : database_path(db_path) {
+DBManager::DBManager(string dbPath) : database_path(std::move(dbPath)) {
```

Comment:
This is the class which saves and uses the `dbPath`, the other classes just pass it down.
two options here:

1. copy the string into a new const string (currently implemented)
2. just use the same string_view as was initialized in `main.h`, since we shouldn't be changing it anyways.

Reply:
looks good, remove std::

Reply:
can't because then the compiler gets confused.